<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nedrysoft::RegExHostMasker::RegExHostMaskerSettingsPageWidget</class>
 <widget class="QWidget" name="Nedrysoft::RegExHostMasker::RegExHostMaskerSettingsPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>456</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nedrysoft::RegExHostMasker::RegExHostMaskerSettingsPageWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="10,0,0">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="expressionsTreeWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="newButton">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="importButton">
       <property name="text">
        <string>Import...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Export...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="enabledLabel">
       <property name="text">
        <string>Enabled:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="enabledCheckBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="descriptionLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="regularExpressionLabel">
       <property name="text">
        <string>Regular Expression:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="regularExpressionLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="substitutionLabel">
       <property name="text">
        <string>Host Substitution:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="substitutionLineEdit"/>
     </item>
     <item row="6" column="1">
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,2,0">
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="matchHopsCheckbox">
         <property name="text">
          <string>Match Hops</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="matchAddressCheckbox">
         <property name="text">
          <string>Match Address</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="matchHostCheckbox">
         <property name="text">
          <string>Match Host</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="maskHostCheckbox">
         <property name="text">
          <string>Mask Host</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="maskAddressCheckbox">
         <property name="text">
          <string>Mask Address</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="addressSubstitutionLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="addressSubstitutionLabel">
       <property name="text">
        <string>Address Substitution:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="hopsLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="hopsLabel">
       <property name="text">
        <string>Hops:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>expressionsTreeWidget</tabstop>
  <tabstop>newButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>importButton</tabstop>
  <tabstop>exportButton</tabstop>
  <tabstop>enabledCheckBox</tabstop>
  <tabstop>descriptionLineEdit</tabstop>
  <tabstop>regularExpressionLineEdit</tabstop>
  <tabstop>substitutionLineEdit</tabstop>
  <tabstop>addressSubstitutionLineEdit</tabstop>
  <tabstop>hopsLineEdit</tabstop>
  <tabstop>matchHopsCheckbox</tabstop>
  <tabstop>matchAddressCheckbox</tabstop>
  <tabstop>matchHostCheckbox</tabstop>
  <tabstop>maskHostCheckbox</tabstop>
  <tabstop>maskAddressCheckbox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
